cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,130 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
```

Comment:
What does this step do?

Reply:
@ibacher it is supposed to reflect the page [actions](http://localhost:8080/actions) where the concept Id is already added.

Reply:
@jwnasambu Surely with a step like this we want to check that the concept already exists in the dictionary rather than to go to the progress notifications page right? This is a "Given" step, so it should be asserting some kind of precondition, e.g.,

```ts
cy.getConcept(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/1000`).should("exist");
```